pecan-waffle.psm1

```diff
@@ -1101,6 +1101,31 @@ function New-PWItem{
 }
 Set-Alias Add-Item New-PWItem -Description 'obsolete: This was added for back compat and will be removed soon'
 
+function Remove-PWTemplate{   
+    [cmdletbinding()]
+    param(
+        [Parameter(Position=1,Mandatory=$true)]
```

Comment:
Can you add `ValueFromPipeline=$true` here? This will enable piping for example 

```powershell
'aspnet-empty`,`aspnet-web` | Remove-PWTemplate
```